src/boost_histogram/_internal/hist.py

```diff
@@ -779,10 +819,123 @@ def __setitem__(self, index, value):
         view[tuple(indexes)] = value
 
     def project(self, *args):
+        # type: (Axis) -> Histogram
         """
-        Project to a single axis or several axes on a multidiminsional histogram.
+        Project to a single axis or several axes on a multidimensional histogram.
         Provided a list of axis numbers, this will produce the histogram over
         those axes only. Flow bins are used if available.
         """
 
         return self._new_hist(self._hist.project(*args))
+
+    # Implementation of PlottableHistogram
+
+    @property
+    def kind(self):
+        # type: () -> Kind
+        """
+        Returns Kind.COUNT if this is a normal summing histogram, and Kind.MEAN if this is a
+        mean histogram.
+
+        :return: Kind
+        """
+        if self._hist._storage_type in {
+            _core.storage.mean,
+            _core.storage.weighted_mean,
+        }:
+            return Kind.MEAN
+        else:
+            return Kind.COUNT
+
+    def values(self, flow=False):
+        # type: (bool) -> ArrayLike
+        """
+        Returns the accumulated values. The counts for simple histograms, the
+        sum of weights for weighted histograms, the mean for profiles, etc.
+
+        If counts is equal to 0, the value in that cell is undefined if
+        kind == "MEAN".
+
+        :param flow: Enable flow bins. Not part of PlottableHistogram, but
+        included for consistency with other methods and flexibility.
+
+        :return: np.ndarray[np.float64]
+        """
+
+        view = self.view(flow)
+        if len(view.dtype) == 0:
+            return view
+        else:
+            return view.value
+
+    def variances(self, flow=False):
+        # type: (bool) -> Optional[ArrayLike]
+        """
+        Returns the estimated variance of the accumulated values. The sum of squared
+        weights for weighted histograms, the variance of samples for profiles, etc.
+        For an unweighed histogram where kind == "COUNT", this should return the same
+        as values if the histogram was not filled with weights, and None otherwise.
+        If counts is equal to 1 or less, the variance in that cell is undefined if
```

Comment:
Perhaps add a note that this condition must be used exactly as described here. counts <= 1 is not the same as counts < 2, when counts returns effective counts.